library/swift/src/BUILD

```diff
@@ -27,9 +29,19 @@ swift_static_framework(
         "UpstreamHttpProtocol.swift",
     ],
     module_name = "Envoy",
-    objc_includes = [
-        "//library/objective-c:EnvoyEngine.h",
-    ],
+    private_deps = ["//library/objective-c:envoy_engine_objc_lib"],
+)
+
+swift_header_collector(
+    name = "collector",
+    deps = ["lib"],
```

Comment:
Does this change make it possible for us to split out the above code into different libraries and still collect them into one framework here?

Reply:
not particularly right here, but overall it depends on how you want to split things up. if you have public API in multiple modules I don't think that will end up working in a single framework, but if you have other internal modules like the EnvoyEngine objc is acting as here now, you could include that in private_deps and have as many as you want. But I think no matter what you'll have to be able to `@_implementationOnly` import them